Themes and impostor box

dhuppenkothen opened the pull request:
Moved the impostor box and shortened the Themes section into a single paragraph. I didn't move anything else into the supplementary material, because I just condensed the language.

arokem commented on supplementary_material.tex:
```diff
@@ -111,6 +111,23 @@ \subsubsection{Hack Sessions}
 At the hack week itself, the time designated for project work will generally be prefaced by a pitch session, where participants with projects give a short description of their project to the group and request help. For junior participants and those new to the format, this can be daunting, even if (or especially when) they do not have a project themselves. In particular junior participants tend to ...
 Because hack weeks are generally longer than typical hackathons (often limited to just one or two days), this will lead to a variety of shorter projects completed within a day or two, and longer projects going on throughout the week. Regular check-in sessions are helpful for pitching new projects, requesting help on projects stuck at a certain point, and re-matching participants after hacks have  ...
 
+\subsection{Box 1: Impostor Syndrome}
```
Maybe remove "Box 1"?

arokem replied:
It would be good to refer to this somewhere in the main text. Maybe a single sentence in the design section pointing to this as an issue, and referring here for more detail?

dhuppenkothen replied:
that's a comment for @aaarendt then, who's our Design section editor. :)

dhuppenkothen commented:
@arokem I fixed the merge conflicts with the design section and added a sentence to the design section I merged from Anthony's PR. Let me know if that addresses your comments?

arokem commented on sections/design.tex:
```diff
@@ -26,4 +26,6 @@ \section*{Design considerations}
-This breadth of possible outcomes makes it difficult to design for all possible participant goals, and calls for adaptive, flexible leadership among hack week organizers.
\ No newline at end of file
+This breadth of possible outcomes makes it difficult to design for all possible participant goals, and calls for adaptive, flexible leadership among hack week organizers.
+The large variety in participant backgrounds and experiences can to inc that many participants will experience impostor syndrome (see also supplementary materials), and it 
```
"can to inc"?

dhuppenkothen replied:
Oops, I should try to write a paper while at a conference. 😳 
Should be fixed now.

arokem commented:
LGTM. Merging!
